templates/systemd.service.j2

```diff
@@ -0,0 +1,26 @@
+# {{ ansible_managed }}
```

Comment:
File naming, "systemd_extra.service.j2" or something similar which has a relation with the acual config vars.